### DISCUSSION TITLE
reboot chia constantly

### DISCUSSION THREAD

**Discussion by claudio10001** [Chia-Network/chia-blockchain#4376] [Support]
chia blockchain closes and when it opens again it asks me for 24 words, it does it 5 or 6 times a day, someone could tell me how to solve it

chia blockchain se cierra y al abrir nuevamente me solicita las 24 palabras, lo hace 5 o 6 veces por dia , algien me podria decir como solucionarlo

**Comment by lvcivs** [comment]
Something must be wrong with your setup. It should only ask for the 24 words if you add a new set of keys, for example after completely uninstalling. You might need to do a full reinstall of the chia software.

Which OS are you on? Did you install from an installer?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
